dev/README

```diff
@@ -1,30 +1,30 @@
-This directory contains informations and tools to help developing the
+This directory contains information and tools to help develop the
                              Coq system
                        ======================
 
 
 Debugging and profiling (in current directory - see doc/debugging.txt)
 -----------------------
 
-ocamldebug-coq: to launch ocaml debugger
+ocamldebug-coq: to launch ocaml debugger (generated by the configure script)
 
 db:      to install pretty-printers from ocaml debugger
-base_db: to install raw pretty-printers from ocaml debugger
+base_db: to install raw pretty-printers from ocaml debugger (subset of db--still needed?)
```

Comment:
The `base_db` is sometimes needed for low-level debugging, e.g. to display de Bruijn indices or internal structures of fixpoints, etc.

In practice, I mostly overwrite the default `constr` printer with `print_pure_constr`, so, it is not absolutely necessary to keep it. Don't know the opinion of others though.